test/test_ff_utils.py

```diff
@@ -435,7 +442,7 @@ def test_stuff_in_queues_integrated(integrated_ff):
     for item in search_res[:8]:
         ff_utils.patch_metadata({}, obj_id=item['uuid'], key=integrated_ff['ff_key'])
     time.sleep(3)  # let queues catch up
-    stuff_in_queue = ff_utils.stuff_in_queues(integrated_ff['ff_env'], check_secondary=True)
+    stuff_in_queue = ff_utils.stuff_in_queues(integrated_ff['ff_env_index'], check_secondary=True)
```

Comment:
``ff_env_index_namespace`` is the name you want.

Reply:
That would be OK.

Reply:
Done.